config/identity/ownership/debug-no-check.json

```diff
@@ -0,0 +1,13 @@
+{
+  "@context": "https://linkedsoftwaredependencies.org/bundles/npm/@solid/community-server/^0.0.0/components/context.jsonld",
+  "@graph": [
+    {
+      "comment": [
+        "DO NOT USE IN PRODUCTION, ONLY FOR DEVELOPMENT, TESTING, OR DEBUGGING.",
+        "Do no verification to determine WebID ownership."
+      ],
```

Comment:
I like this comment style for such cases.

Reply:
Well you should since it's you who suggested it 😂